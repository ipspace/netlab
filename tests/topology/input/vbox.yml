provider: virtualbox
defaults:
  inventory: dirs

nodes:
- name: c_nxos
  device: nxos
- name: a_eos
  device: eos
- name: a_eos_2
  device: eos

links:
- c_nxos:
  a_eos:
  a_eos_2:

- c_nxos:
  a_eos:
