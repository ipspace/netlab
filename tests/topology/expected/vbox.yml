input:
- topology/input/vbox.yml
- package:topology-defaults.yml
links:
- bridge: input_1
  interfaces:
  - ifindex: 1
    ifname: Ethernet1/1
    ipv4: 172.16.0.1/24
    node: c_nxos
  - ifindex: 1
    ifname: Ethernet1
    ipv4: 172.16.0.2/24
    node: a_eos
  - ifindex: 1
    ifname: Ethernet1
    ipv4: 172.16.0.3/24
    node: a_eos_2
  linkindex: 1
  node_count: 3
  prefix:
    ipv4: 172.16.0.0/24
  type: lan
- interfaces:
  - ifindex: 2
    ifname: Ethernet1/2
    ipv4: 10.1.0.2/30
    node: c_nxos
  - ifindex: 2
    ifname: Ethernet2
    ipv4: 10.1.0.1/30
    node: a_eos
  linkindex: 2
  node_count: 2
  prefix:
    ipv4: 10.1.0.0/30
  type: p2p
name: input
nodes:
  a_eos:
    af:
      ipv4: true
    box: arista/veos
    device: eos
    id: 2
    interfaces:
    - bridge: input_1
      ifindex: 1
      ifname: Ethernet1
      ipv4: 172.16.0.2/24
      linkindex: 1
      name: a_eos -> [c_nxos,a_eos_2]
      neighbors:
      - ifname: Ethernet1/1
        ipv4: 172.16.0.1/24
        node: c_nxos
      - ifname: Ethernet1
        ipv4: 172.16.0.3/24
        node: a_eos_2
      type: lan
    - ifindex: 2
      ifname: Ethernet2
      ipv4: 10.1.0.1/30
      linkindex: 2
      name: a_eos -> c_nxos
      neighbors:
      - ifname: Ethernet1/2
        ipv4: 10.1.0.2/30
        node: c_nxos
      type: p2p
    loopback:
      ipv4: 10.0.0.2/32
    mgmt:
      ifname: Management1
      ipv4: 192.168.121.102
      mac: 08:4f:a9:00:00:02
    name: a_eos
  a_eos_2:
    af:
      ipv4: true
    box: arista/veos
    device: eos
    id: 3
    interfaces:
    - bridge: input_1
      ifindex: 1
      ifname: Ethernet1
      ipv4: 172.16.0.3/24
      linkindex: 1
      name: a_eos_2 -> [c_nxos,a_eos]
      neighbors:
      - ifname: Ethernet1/1
        ipv4: 172.16.0.1/24
        node: c_nxos
      - ifname: Ethernet1
        ipv4: 172.16.0.2/24
        node: a_eos
      type: lan
    loopback:
      ipv4: 10.0.0.3/32
    mgmt:
      ifname: Management1
      ipv4: 192.168.121.103
      mac: 08:4f:a9:00:00:03
    name: a_eos_2
  c_nxos:
    af:
      ipv4: true
    box: cisco/nexus9300v
    device: nxos
    id: 1
    interfaces:
    - bridge: input_1
      ifindex: 1
      ifname: Ethernet1/1
      ipv4: 172.16.0.1/24
      linkindex: 1
      name: c_nxos -> [a_eos,a_eos_2]
      neighbors:
      - ifname: Ethernet1
        ipv4: 172.16.0.2/24
        node: a_eos
      - ifname: Ethernet1
        ipv4: 172.16.0.3/24
        node: a_eos_2
      type: lan
    - ifindex: 2
      ifname: Ethernet1/2
      ipv4: 10.1.0.2/30
      linkindex: 2
      name: c_nxos -> a_eos
      neighbors:
      - ifname: Ethernet2
        ipv4: 10.1.0.1/30
        node: a_eos
      type: p2p
    loopback:
      ipv4: 10.0.0.1/32
    mgmt:
      ifname: mgmt0
      ipv4: 192.168.121.101
      mac: 08:4f:a9:00:00:01
    name: c_nxos
provider: virtualbox
